agent/tests/test_pipelines/test_2/test_zabbix_http.py

```diff
@@ -27,3 +28,17 @@ def test_stop(self, cli_runner, name=None, check_output_file_name=None):
 
     def test_output_schema(self, name=None, pipeline_type=None, output=None):
         pytest.skip()
+
+    def test_start(self, cli_runner, name, sleep):
+        super().test_start(cli_runner, name, sleep)
+
+    def test_output(self, name, pipeline_type, output):
+        def compare_output():
```

Comment:
I think you can use `super().test_output` here instead of duplicating the code

Reply:
Agree. Fixed